README.md

```diff
@@ -732,12 +732,22 @@ Output:
 
 If the optional `EBUR128` property is set to `true`, several loudness measurements will be taken based on the [EBU R 128](https://en.wikipedia.org/wiki/EBU_R_128) standard. Given that this takes significantly longer than the rest of the inspection task, when submitting jobs that include loudness measurement, you may want to include two `Inspect` tasks, so that one can return results more quickly. `EBUR128` measurements are intended for audio and video source files.
 
+Inspect can also selectively return metadata from the file. This only currently works for audio files using `ffprobe -show_entries format_tags` which returns whatever tags are detected in the file metadata. Metadata can be matched using the `IncludeMetadata` object, either on tag `Keys` or `Values`. Currently only `StringIncludes` is supported, which allows specifying a value to match as a regular expression against the tag key or value.
```

Comment:
For consistency, this could say:

> Metadata can be matched using the optional `IncludeMetadata` property

I'm sure there are inconsistencies in the doc, but I try to always state explicitly when something is optional, I think I've been trying to use _property_ over _object_, because I had been using them pretty interchangeably and decided to pick on.

Reply:
I used `object` as it was a more complex value specified, not a simple value.